galaxy/wrapper/plotHeatmap.xml

```diff
@@ -134,6 +140,10 @@
             <when value="yes">
                 <expand macro="sortRegions" />
                 <expand macro="sortUsing" />
+                <param argument="--sortUsingSamples" type="text" size="30"
```

Comment:
size can be removed it is not used anymore

Reply:
then I remove it from other options too